pandas/core/strings.py

```diff
@@ -1090,7 +1090,11 @@ def _wrap_result_expand(self, result, expand=False):
         else:
             index = self.series.index
             if expand:
-                cons_row = self.series._constructor
+                def cons_row(x):
+                    if x is np.nan:
```

Comment:
I initially tried that, but `np.isnan` can't handle strings. `x` here is either NaN or a list of strings. I also only want that conditional to be true if `x` is a single null value. I agree that `is np.nan` isn't ideal: what would be the better way of doing this?